packages/docs-app/src/examples/core-examples/buttonGroupExample.tsx

```diff
@@ -29,102 +29,76 @@ import {
-        fill: false,
-        iconOnly: false,
-        intent: Intent.NONE,
-        large: false,
-        minimal: false,
-        outlined: false,
-        vertical: false,
-    };
-
-    private handleFillChange = handleBooleanChange(fill => this.setState({ fill }));
-
-    private handleIconOnlyChange = handleBooleanChange(iconOnly => this.setState({ iconOnly }));
-
-    private handleIntentChange = (intent: Intent) => this.setState({ intent });
-
-    private handleLargeChange = handleBooleanChange(large => this.setState({ large }));
-
-    private handleMinimalChange = handleBooleanChange(minimal => this.setState({ minimal }));
-
-    private handleOutlinedChange = handleBooleanChange(outlined => this.setState({ outlined }));
-
-    private handleVerticalChange = handleBooleanChange(vertical => this.setState({ vertical }));
-
-    public render() {
-        const { iconOnly, intent, ...bgProps } = this.state;
-        // props for every button in the group
-        const buttonProps = { intent };
-
-        const intentLabelInfo = (
-            <Tooltip
-                content={
-                    <span className={Classes.TEXT_SMALL}>
-                        Intents are set individually on each button <br />
-                        in the group, not the ButtonGroup wrapper.
-                    </span>
-                }
-                placement="top"
-                minimal={true}
+export const ButtonGroupExample: React.FC<ExampleProps> = props => {
+    const [alignText, setAlignText] = React.useState<Alignment>(Alignment.CENTER);
+    const [fill, setFill] = React.useState(false);
+    const [iconOnly, setIconOnly] = React.useState(false);
+    const [intent, setIntent] = React.useState<Intent>(Intent.NONE);
+    const [large, setLarge] = React.useState(false);
+    const [minimal, setMinimal] = React.useState(false);
+    const [outlined, setOutlined] = React.useState(false);
+    const [vertical, setVertical] = React.useState(false);
+
+    const options = (
+        <>
+            <H5>Props</H5>
+            <Switch checked={fill} label="Fill" onChange={handleBooleanChange(setFill)} />
+            <Switch checked={large} label="Large" onChange={handleBooleanChange(setLarge)} />
+            <Switch checked={minimal} label="Minimal" onChange={handleBooleanChange(setMinimal)} />
+            <Switch checked={outlined} label="Outlined" onChange={handleBooleanChange(setOutlined)} />
+            <Switch checked={vertical} label="Vertical" onChange={handleBooleanChange(setVertical)} />
+            <IntentSelect intent={intent} label={intentLabelInfo} onChange={setIntent} />
+            <AlignmentSelect align={alignText} onChange={setAlignText} />
+            <H5>Example</H5>
+            <Switch checked={iconOnly} label="Icons only" onChange={handleBooleanChange(setIconOnly)} />
+        </>
+    );
+
+    return (
+        <Example options={options} {...props}>
+            <ButtonGroup
+                fill={fill}
```

Comment:
```suggestion
                alignText={alignText}
                fill={fill}
```

The align text prop is not wired up

Reply:
Thanks for the catch! 00529d2b714ad07f014e4e091fa9cae43221b6b2